parser-typechecker/src/Unison/Codebase/IntegrityCheck.hs

```diff
@@ -198,19 +194,15 @@ integrityCheckAllBranches = do
           Q.loadBranchObjectIdByCausalHashId causalHashId >>= \case
             Nothing -> do
               ch <- Q.expectHash (DB.unCausalHashId causalHashId)
-              failure $ "Expected branch object for causal hash ID: " <> pShow causalHashId
+              logError $ "Expected branch object for causal hash ID: " <> pShow causalHashId
               pure (Set.singleton $ MissingObjectForChildCausal ch)
             Just foundBranchId
               | foundBranchId /= branchObjId -> do
-                  failure $ "Expected child branch object to match canonical object ID for causal hash's namespace: " <> pShow (causalHashId, foundBranchId, branchObjId)
+                  logError $ "Expected child branch object to match canonical object ID for causal hash's namespace: " <> pShow (causalHashId, foundBranchId, branchObjId)
                   ch <- Q.expectHash (DB.unCausalHashId causalHashId)
                   pure (Set.singleton $ MismatchedObjectForChild ch branchObjId foundBranchId)
               | otherwise -> pure mempty
 
-failure :: TL.Text -> Sqlite.Transaction ()
```

Comment:
Unrelated change, I just noticed that we had a `failure = logError`, so I deleted it all calls with `logError`.